finqwerty/src/main/res/raw/pro1_qwerty_svk_1.kcm

```diff
@@ -0,0 +1,329 @@
+###############################################################
+#                                                             #
+# Slovak (QWERTY) layout                                      #
+# based on http://ascii-table.com/keyboard.php/245            #
+#                                                             #
+###############################################################
+
+type OVERLAY
+
+# both yellow-arrow keys are 464 (FUNCTION)
+
+#map key 12 PLUS (EQUALS?)
+#map key 13 GRAVE
+map key 111 PLUS
+
+map key 41 Q
+map key 16 W
+map key 17 E
+map key 18 R
+map key 19 T
+map key 20 Y
+map key 21 U
+map key 22 I
+map key 23 O
+map key 24 P
+map key 38 SEMICOLON
+
+map key 26 BACKSLASH
+map key 43 A
+map key 30 S
+map key 31 D
+map key 32 F
+map key 33 G
+map key 34 H
+map key 35 J
+map key 36 K
+map key 37 L
+map key 40 APOSTROPHE
+
+map key 25 LEFT_BRACKET
+map key 39 RIGHT_BRACKET
+map key 27 Z
+map key 44 X
+map key 45 C
+map key 46 V
+map key 47 B
+map key 48 N
+map key 49 M
+map key 50 COMMA
+map key 51 PERIOD
+map key 52 SLASH
+
+# orig BACK
+#map key 158 ESCAPE
```

Comment:
I think we prefer escape key to actually be ESCAPE, so I'd keep this in.

But reportedly it was going to be changed to ESCAPE in the stock layout too, so maybe this bit is unnecessary now. We'll see when we get the actual devices.

Reply:
I think this one is resolved by now :)